license

```diff
@@ -0,0 +1,21 @@
+The MIT License
```

Comment:
I'll also update README.md with instructions on how to contribute once I fully understand how CLA assistant works.